<launch>

    <include file="$(find sensors)/launch/usv_simulation.launch" />

    <include file="$(find sensors)/launch/pid.launch" />

    <node pkg="sensors" type="trial_2x5.py" name="trial_2x5" />

</launch>
